plugins/modules/htpasswd.py

```diff
@@ -41,8 +41,11 @@
     description:
       - Encryption scheme to be used.  As well as the four choices listed
         here, you can also use any other hash supported by passlib, such as
-        md5_crypt and sha256_crypt, which are linux passwd hashes.  If you
-        do so the password file will not be compatible with Apache or Nginx
+        portable_apache22 and host_apache24; or md5_crypt and sha256_crypt,
+        which are linux passwd hashes.  Only some schemes in addition to
+        the four choices below will be compatible with Apache or Nginx and
+        supported schemes depend on passlib version and its dependencies
+      - See U(https://passlib.readthedocs.io/en/stable/lib/passlib.apache.html#passlib.apache.HtpasswdFile) parameter default_scheme.
```

Comment:
```suggestion
      - See U(https://passlib.readthedocs.io/en/stable/lib/passlib.apache.html#passlib.apache.HtpasswdFile) parameter C(default_scheme).
```

Reply:
Sounds better. I had given it some thought but decided to just follow the existing (non-)formatting. Thanks!